### ISSUE TITLE
[Bug] Rider Positioning

### ISSUE THREAD

**Issue by Crowguard** [Saibot393/Rideable#84]
The `Mounting Position` option seems to be having the tokens just remain stationary instead of moving with the tile when it is rotated via MATT's Rotation action. Works as expected when rotated manually or when using a loop of MATT Alter action.

https://github.com/user-attachments/assets/8bac0b3f-1754-4d93-b606-4f160bee3ff6

Foundry 12.330
MATT 12.02
Rideable 3.0.5

**Comment by Saibot393** [comment]
Everything seems to work correctly in my test world. The only way i can replicate the shown behaviour is when i deactivate the "link rotations" rideable setting. Are there any errors in the console? If not, could you either send me screenshots of the rideable world settings or send me the files of the world you used for testing?

**Comment by Crowguard** [comment]
"Link Rotations" is enabled. 

![image](https://github.com/user-attachments/assets/a9dffea5-3aed-4ad1-b69a-c5bad5dc45a9)


Couple of warnings I missed the first time.

![Foundry_Virtual_Tabletop_9dHeTU0lac](https://github.com/user-attachments/assets/64ca2ed8-5096-45b8-99b3-6f107e191985)


https://github.com/user-attachments/assets/c84ea8c3-8c7e-42c7-bb9b-23c39c647146

**Comment by Saibot393** [comment]
I just noticed, that the mountingis also done via a MATT action. Could you try mounting the token manually and then activate the rotation action. If that works, adding a small delay between mounting and rotating should probably allow for the script to work. If that is the case i will look into a fix so that the current version also works

**Comment by Crowguard** [comment]
Yep. That seems to be doing it. Either of those works as expected.

**Comment by Saibot393** [comment]
Perfect, then i think i know how to fix it

**Comment by Rukasoar** [comment]
I'm not sure if this is entirely related, but I'm having a bug with the Lock Rotation feature, and it persists between "mount by move" as well as manual mounting. I put a video of it on [Imgur](https://imgur.com/a/foundry-vtt-rideable-bug-Qw8KgiP) as I was having problems uploading the video here. I also attempted embedding it below, but unsure why it didn't work right. I very rarely use this site so I'm unsure how to post it in a way you can view it here, apologies.

<blockquote class="imgur-embed-pub" lang="en" data-id="a/Qw8KgiP"><a href="//imgur.com/a/Qw8KgiP">Foundry VTT - Rideable bug</a></blockquote>

Is there a setting I am missing or is this a bug?
Info: I am using Rideable V 3.2.3 & 3.2.4 (Tested both as latest update did something with rotation
Pf2e V 6.9.0
Attempted Auto Rotate Module with no success
No modules currently interacting with tokens
No Error with console appears during the linked video

**Comment by Saibot393** [comment]
When you tested this, were you logged in as a GM or a Player, if you were logged in as a player, was a GM account also logged in at the same time?

**Comment by Rukasoar** [comment]
I was using a test player on a seperate window, while I hosted it as a GM which is the viewpoint you see.

Both GM and Player were online, and I controlled the tokens with the player account. Although, if it matters, it did the same when I controlled it with the GM account, too.

**Comment by Saibot393** [comment]
Are there any errors in either the players or the GMs console (F12)? Also, could you send me a screenshot of the ridden tokens Rideable settings?

**Comment by Rukasoar** [comment]
I added a video to youtube to share, unsure why Imgur was compressing everything I was uploading. Again, I rarely use this site so I'm unsure why I can't upload anything. In the video I show the console, which shows no errors, as well as both the token (vehicle) settings and the module settings. I did it the same way as before, and controlled the player on a separate window.

Also, thank you for your time in addressing this, this is the second to last module I need to get working for my game.

https://youtu.be/20PX_YF60rQ

**Comment by Saibot393** [comment]
I tested a bit with the settings you have shown in the video, but everything seems to work correctly for me. Which foundry Version are you using, v12?
Have you tried running [Find the Culprit!](https://foundryvtt.com/packages/find-the-culprit/) to see if it may be a compatibility issue?

**Comment by Rukasoar** [comment]
First off, thank you for showing me "find the Culprit," I've never seen that module anywhere and I feel like I can solve many of my problems on my own in the future.

lastly, this module helped me find the issue: Rideable is not compatible with Bailey Wiki's Mass Edit. Not sure why, since all it does is create a framework to copy and paste settings between tokens and tiles, but I guess it throws off the rotation in your module. I can run my game without it, so I'll uninstall it in favor of Rideable. 

Thank you again for your help, and definitely thank you for both your module and the one you linked me to.

**Comment by Saibot393** [comment]
Good to hear that it is solved for now. I will look into the module incompatibility to see if i can fix the bug
